qemu/tests/watchdog.py

```diff
@@ -436,7 +436,11 @@ def heartbeat_test():
 
     # main procedure
     test_type = params.get("test_type")
-    check_watchdog_support()
+    watchdog_device_type = params.get("watchdog_device_type")
+    if watchdog_device_type == "itco":
+        pass
```

Comment:
What's the supported version for this type ？

Reply:
You mean the qemu-kvm version?

Reply:
Yes,  or the product version ?

Reply:
The product version  is rhel9.3.0.
The qemu-kvm version support is qemu-kvm-8.0

Reply:
Add qemu-kvm version support issue for this code:
    if watchdog_device_type == "itco":
        **if compare_qemu_version(8, 0, 0, is_rhev=False) is True:**
            pass

@yanan-fu , What do you think？

Reply:
So, please update the case configuration to limit the support version.

Reply:
I have updated this issue to cfg, please check it, thanks